src/components/ErrorBoundary/__tests__/ErrorBoundary-test.js

```diff
@@ -16,41 +16,43 @@ describe('components/ErrorBoundary', () => {
         wrapper.find(WrappedComponent).simulateError(wrappedError);
     };
 
-    describe('should render', () => {
+    describe('render()', () => {
         test('the wrapped component when no error is thrown', () => {
             const wrapper = getWrapper();
 
             expect(wrapper).toMatchSnapshot();
         });
 
-        test('nothing by default when an error is thrown', () => {
+        test('should render default error component an error is thrown', () => {
             const wrapper = getWrapper();
 
             simulateError(wrapper);
 
-            expect(wrapper).toMatchSnapshot();
+            expect(wrapper.find('DefaultError').exists()).toBe(true);
         });
 
         test('the component specified when an error is thrown', () => {
+            const ErrorComponent = () => <div>Error</div>;
             const wrapper = getWrapper({
-                component: <div>Error</div>,
+                errorComponent: ErrorComponent,
             });
 
             simulateError(wrapper);
 
-            expect(wrapper).toMatchSnapshot();
+            expect(wrapper.find(ErrorComponent).exists()).toBe(true);
```

Comment:
Does looking at the snapshot not assert this already?

Reply:
It does, but I find snapshot tests to be very brittle and it is not explicit what your output should be.

I normally wouldn't have touched this assertion, but this functionality was not working when I tested with a custom component (would only work with a dom node not a react component)